training/main/scripts/create_kb.py

```
@@ -24,7 +23,7 @@
 def main(
     trf_name: str,
     save_path_kb: Path = project_path / "assets/knowledge_bases/knowledge_base.kb",
-    langs_to_fetch: List[str] = ["da", "en"],
+    langs_to_fetch: list[str] = ["da", "en"],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)